src/assets/scss/components/_oktaSigninWidget.scss

```
@@ -157,23 +170,36 @@ sign-up-modal {
 
     &:first-of-type {
 
-      input[type=text]:not(.chzn-search input),
-      input[type=password],
-      .chzn-container a {
+      input:not(.chzn-search input), .chzn-container a {
         border-top-left-radius: 6px;
         border-top-right-radius: 6px;
       }
     }
 
     &:last-of-type {
-
-      // We don't include password as it's the only field on it's step and has the password requirements below it.
-      input[type=text]:not(.chzn-search input),
-      .o-form-input-error,
-      .chzn-container a {
+      // Usually the last fieldset has rounded bottom borders
+      // ".chzn-container a" is for select boxes
+      input:not(.chzn-search input), .o-form-input-error, .chzn-container a {
         border-bottom-left-radius: 6px;
         border-bottom-right-radius: 6px;
       }
+
+      // However, if it is a password field with a password requirements sibling, the password
+      // requirements box gets the rounded border instead of the input or error message
+      &[data-se="o-form-fieldset-credentials.passcode"]:has(+ [data-se="password-authenticator--rules"]) {
+        input:not(.chzn-search input), .o-form-input-error {
```

I don't think you need this here as it won't have a dropdown.
```suggestion
        input, .o-form-input-error {
```

---

Speaking of the `:not(.chzn-search input)`, I don't think that it is needed anywhere. I made a screen recording with audio so I could explain what I was seeing. But I'm thinking we can drop the `:not(.chzn-search input)` on all of these selectors because `.chzn-search input` never selects anything visible to the user. But maybe I'm missing something.

https://github.com/user-attachments/assets/d698b4dd-687d-4ae1-88ba-1ee3d05fc088

---

You're correct; we can remove it. Apart from line 199, we need to have the selector there
```
input[type=text]:not(.chzn-search input
```

---

Why is the one on line 199 needed? Everything is still looking good without it.

---

When testing it locally, the account type select input was styled incorrectly.

---

I'm still not seeing any issues with account type selector, but do you mind sharing a screenshot, and I'll add the `:not` part of the selector back?

---

I think we're on about different lines, as it's still included in your PR. It was on line `199` at one point, but now it's changed to `227`.

https://github.com/CruGlobal/give-web/blob/bc9398ecb8683b0d0cce9bfdf87d5455b6cf1b9f/src/assets/scss/components/_oktaSigninWidget.scss#L227

---

Ah, sorry for any misunderstanding. I'm glad we're both on the same page.